README.md

```diff
@@ -1,5 +1,95 @@
-# QR Code detection package
+# QR Code Scanner Node
```

Comment:
Nice Readme! But i am not too sure if documenting source code in the repo os necessary when we are using doxygen/docstring commenting anyway (which you are doing very well), as it only creates duplicate data that has to be kept in sync.

Reply:
as wrote in the commit message, the readme is partially generated, so not to much effort went into this. but your point still stands ;)